src/appComponents/twitterwall/singletweet.js

```diff
@@ -14,22 +14,22 @@ function tweet($timeout,$rootScope) {
         array:"="
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Missing space before value for key "array".](https://www.codacy.com/app/triplez/open-event-webapp/file/2443856941/issues/source?bid=3233857&fileBranchId=3245777#l14)